apps/central-scan/backend/src/fujitsu_scanner.ts

```diff
@@ -150,19 +150,29 @@ export class FujitsuScanner implements BatchScanner {
       args.push('--endorser-string', `${imprintIdPrefix}_%04ud`);
     }
 
-    const MM_PER_INCH = 25.3967;
+    /**
+     * We've occasionally seen the Fujitsu return images where HMPB timing marks are ever so
+     * slightly clipped, so we add some buffer to the page height to ensure that we capture
+     * everything. Scanning extra is safe because the HMPB interpreter trims any extra scan area.
+     */
+    const HMPB_HEIGHT_BUFFER_INCHES = 0.25;
+    const MM_PER_INCH = 25.4;
```

Comment:
I'm not sure why we'd previously set the mm per inch conversion factor to 25.3967, but I did confirm that the conversion factor is in fact exactly 25.4. This difference is so small that changing it alone doesn't solve the larger clipping issue.